docker/launch_prebuilding.sh

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env bash
+set -euo pipefail
+
+# Parse input params
+python_version=${1:-lastest}
+cuda_version=${2:-11.8}
+gpu_backend=${3:-cuda}
+
+export FF_CUDA_ARCH=all
+export BUILD_LEGION_ONLY=ON
+
+# Build Docker Flexflow Container
+echo "building docker"
+./docker/build.sh flexflow
+export detached=ON
+./docker/run.sh flexflow
+
+# Copy legion libraries to host
+echo "copy legion libaries"
+echo "flexflow-${gpu_backend}-${cuda_version}:${python_version}"
+container_id=$(docker ps | grep flexflow-cuda-11.8:latest | awk '{print $1}')
+echo "$container_id"
+docker cp "$container_id":/usr/FlexFlow/build/deps ~/buildlegion
```

Comment:
here, we can copy the installed legion library from the installation path (build/export/legion), so we don't need to copy all the build files from all the dependencies